eb

```diff
@@ -60,7 +60,7 @@ PYTHON=
 # - EB_INSTALLPYTHON is set when EasyBuild is installed as a module (by EasyBuild). It is set to the PYTHON
 #   used during that installation (for example, you could override PYTHON using EB_PYTHON at installation
 #   time, this variable preserves that choice).
-for python_cmd in "${EB_PYTHON}" "${EB_INSTALLPYTHON}" 'python' 'python3' 'python2'; do
+for python_cmd in "${EB_PYTHON}" "${EB_INSTALLPYTHON}" 'python' 'python3'; do
```

Comment:
Maybe switch `python3` and `python`?

That's important especially on CentOS 7.x, which is still quite popular (~50% according to latest EasyBuild User Survey)

Reply:
60266d5